app/javascript/utils/Navigation.js

```diff
@@ -3,11 +3,11 @@
  * @param {Boolean} firstParam - whether the nav param is the first query param in the URL
  */
 function navQueryParam(workflow, firstParam) {
-  return workflow ? `${firstParam ? '?' : '&'}nav=${workflow}` : "";
```

Comment:
these all look like unrelated prettier file changed.  this PR is so big can we do that in a separate PR?

Reply:
sorry, don't have time to go through and undo all of them, there are more important things I need to address for this PR and I don't think changing quotes and removing a space are too bad